src/map/clif.cpp

```diff
@@ -17648,6 +17653,10 @@ void clif_parse_configuration( int32 fd, map_session_data* sd ){
 
 			sd->hd->homunculus.autofeed = flag;
 			break;
+		case CONFIG_DISABLE_SHOWCOSTUMES:
+			sd->status.disable_showcostumes = !flag;
```

Comment:
```suggestion
			sd->status.disable_showcostumes = flag;
```
No need to invert anymore after fixing the default value.